### PR TITLE
changed favicon and title to reflect the developer dao logo

### PR DESCRIPTION
**Pull request by nheingit**
This changes the favicon and title to reflect the developer dao logo, and changes the title from "react app" to "Developer Dao"

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Developer Dao</title>
```
Comment (with-heart) [inline public/index.html:27]:
```suggestion
    <title>Developer DAO</title>
```

Just for consistency across platforms. It's already named that way on Discord and here on GitHub.

**Review by with-heart — APPROVED** [review]
Nice little improvement!

File: public/index.html (hunk 1 above)
Comment (nheingit) [reply public/index.html]: Done! Nice catch.

**Review by aesthytik — APPROVED** [review]
good work

**Pull request merged**